ironfish/src/primitives/note.test.ts

```diff
@@ -0,0 +1,48 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at https://mozilla.org/MPL/2.0/. */
+import { createNodeTest, useAccountFixture, useMinerBlockFixture } from '../testUtilities'
+
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at https://mozilla.org/MPL/2.0/. */
+describe('Note', () => {
+  const nodeTest = createNodeTest()
+
+  it('check if a transaction is not a miners fee', async () => {
+    const nodeA = nodeTest.node
+
+    // Create an account A
+    const accountA = await useAccountFixture(nodeA.wallet, () =>
+      nodeA.wallet.createAccount('testA'),
+    )
+    const accountB = await useAccountFixture(nodeA.wallet, () =>
+      nodeA.wallet.createAccount('testB'),
+    )
+
+    // Create a block with a miner's fee
+    const block1 = await useMinerBlockFixture(nodeA.chain, 2, accountA)
+    await nodeA.chain.addBlock(block1)
+    await nodeA.wallet.updateHead()
+
+    const transaction = await nodeA.wallet.createTransaction(
+      accountA,
+      [
+        {
+          publicAddress: accountB.publicAddress,
+          amount: BigInt(1),
+          memo: '',
+        },
+      ],
+      BigInt(1),
+      0,
+    )
+    const encryptedNote = transaction.getNote(0)
+    const note = encryptedNote.decryptNoteForOwner(accountB.incomingViewKey)
+    // TODO(joe): unhardcode address in circuit updates and update this test
+    expect(note?.senderAddress()).toBe(
+      '8a4685307f159e95418a0dd3d38a3245f488c1baf64bc914f53486efd370c563',
+    )
+    // expect(note?.senderAddress()).toBe(accountA.publicAddress)
+  }, 500000)
```

Comment:
```suggestion
  })
```

We generally group tests on the amount of time needed to run after fixtures are generated, so if a test needs more than the default, it probably should live in a `slow` test file. This one shouldn't need to with a tx fixture, though

Reply:
nice catch